libgalois/src/analytics/louvain_clustering/louvain_clustering.cpp

```diff
@@ -352,8 +360,11 @@ LouvainClusteringWithWrap(
       std::is_integral_v<EdgeWeightType> ||
       std::is_floating_point_v<EdgeWeightType>);
 
+  std::vector<std::string> temp_node_property_names = {
+      "CurrentId", "PreviousId", "DegreeWt"};
```

Comment:
These names need to have a random number added to them to avoid name conflicts with existing user properties. Or you could prefix them with something like `_katana_` and document that we reserve all property names which begin with that.